app/main/controller/image_ocr_controller.py

```diff
@@ -37,7 +38,7 @@ def post(self):
             return
 
         app.logger.info(
-            f"[Alegre OCR] [image_uri {image.source.image_uri}] Image OCR response package looks like {response}")
+            f"[Alegre OCR] [image_uri {image.source.image_uri}] Image OCR response package looks like {json.dumps(response)}")
```

Comment:
This seems like it is mostly for debugging, right, as it is logging the full response instead of just the text?

Reply:
we want to see the bounding_poly in the ocr response
gonna do a research task based on that. to determine what percentage of the image has to be text in order to be considered for ocr only.